README.md

```diff
@@ -19,7 +19,7 @@ Here is what I am using:
 rbenv 1.1.2
```

Comment:
Just thinking out loud... once you get your blog post up, it might make sense to link to the post from this file, but delete the tutorial steps from this file at the same time.